### PR TITLE
jsdialog: fix menubutton alignment with other controls in grids

### PR DESCRIPTION
**Pull request by caolanm**
The broad button selector in btns.css matched jsdialog menubuttons, overriding their margin and width. This caused menubuttons to be offset by 5px from comboboxes and listboxes in the same grid column, since those are div elements that do not match the btns.css selector.

Exclude .menubutton from the btns.css catch-all so menubuttons only get their jsdialogs.css styling. Also fix the menubutton rule to use the standard control height (32px), font size, and padding to match other controls, start-align the content, and push the dropdown arrow to the end of the button.

Add a regression test that asserts the menubutton and listbox-container share the same left edge in the Border tab of the Character dialog.

Change-Id: I3ca1a8c46f905f849d08d087f79ad940d8e2ee57

* Resolves: # 
* Target version: main

### TODO

- [ ] ...

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by smehrbrodt** [comment]
This fixes the alignment of the color boxes in the sparkline dialog too, good.

However, the text seems to have a fixed width now - longer text should expand the widget, not break into multiple lines.

Bildschirmfoto vom 2026-03-26 16-13-19

**Comment by caolanm** [comment]
That new commit solves the text wrapping issue I believe

**Comment by smehrbrodt** [comment]
Looks good now, thanks!

**Review by pedropintosilva — APPROVED** [review]
This also fixed the disable status:

Before

Image

after

Image

**Comment by pedropintosilva** [comment]
conflicts solved

File: browser/css/jsdialogs.css [hunk 1]
```diff
@@ -1969,17 +1969,19 @@ img.context-menu-icon {
 /* menu button */
 .jsdialog.menubutton {
 	box-sizing: border-box;
+	height: 32px;
 	line-height: normal;
-	font-size: calc(var(--default-font-size)*1.2);
+	width: max-content;
+	font-size: var(--default-font-size);
```
Comment (pedropintosilva) [inline browser/css/jsdialogs.css:1975]: There is a change of font-size here seems better, but I don't know who added the previous value and if there was any magic reason behind it.

**Review by pedropintosilva — COMMENTED** [review]
probably some visual tests (the ones with image diff) need adjusting? such as desktop/impress/sidebar_spec.js:39 see baseline image esktop/impress/sidebar_spec.js/sidebar_menubutton_color.png  .

**Comment by caolanm** [comment]
I'll restore this via gerrit, though I think samuel+henry may have already done this
